db/migrate/20230614045753_add_fields_to_feed.rb

```diff
@@ -0,0 +1,12 @@
+class AddFieldsToFeed < ActiveRecord::Migration[6.1]
+  def change
+    add_reference :feeds, :saved_search, index: true
+    add_reference :feeds, :user, index: true
+    add_reference :feeds, :team
+    add_column :feeds, :description, :text
+    add_column :feeds, :tags, :string
```

Comment:
Question: Why `string` and not `jsonb`? Or, why not `array`, like `licenses`?

Reply:
`string` as I implemented this field same as `send_every` https://github.com/meedan/check-api/blob/develop/db/migrate/20230516204030_create_tipline_newsletters.rb#L16